src/test/java/de/retest/recheck/ui/descriptors/RootElementFilterTest.java

```diff
@@ -0,0 +1,314 @@
+package de.retest.recheck.ui.descriptors;
+
+import static de.retest.recheck.ui.Path.fromString;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import java.util.List;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import de.retest.recheck.ignore.CompoundFilter;
+import de.retest.recheck.ignore.Filter;
+import de.retest.recheck.review.ignore.AttributeFilter;
+import de.retest.recheck.ui.Path;
+import de.retest.recheck.ui.image.Screenshot;
+import de.retest.recheck.ui.image.Screenshot.ImageType;
+
+class RootElementFilterTest {
+
+	RootElementFilter rootElementFilter;
+
+	@Nested
+	class AttributeFilterTests {
+		private final Element element = mock( Element.class );
+		Filter filter;
+		Attributes state;
```

Comment:
Hmmm, I find "state" confusing. Is there a reason why you didn't name it "attributes"?